js/components/CategoriesTable.jsx

```diff
@@ -18,6 +18,12 @@ var CategoriesTable = React.createClass({
              message: null };
   },
 
+  componentWillReceiveProps: function(newProps) {
```

Comment:
Fixes the issue with the "just added a category" message staying after the user navigates to a different profile page.